### DISCUSSION TITLE
[Feature request] Add warning level to the items

### DISCUSSION THREAD

**Discussion by oxigen-dev** [sysadminsmedia/homebox#742] [Ideas]
### What is the problem you are trying to solve with this feature?

Currently there is no warning level for the items in the inventory.

### What is the solution you are proposing?

I'd propose to add a warning level (quantity) to the items and also an API which can provide the items which already reached its warning level. With this, it would be possible with an external application to add those items to a shopping list.

### Contributions

- [X] I have searched through existing issues and feature requests to see if my idea has already been proposed.
- [ ] If this feature is accepted, I would be willing to help implement and maintain this feature.
- [ ] If this feature is accepted, I'm willing to sponsor the development of this feature.

**Comment by katosdev** [comment]
Seems like a cool idea, thanks for the suggestion.
Throwing this open as help wanted by the community 😄

**Comment by tankerkiller125** [comment]
Please note we're converting this issue into a discussion as per our new procedures/policies. You can read more about that in our announcement #712
